orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/kato/tasks/JarDiffsTaskSpec.groovy

```diff
@@ -103,7 +113,7 @@ class JarDiffsTaskSpec extends Specification {
     Response jarsResponse = new Response('http://foo.com', 200, 'OK', [], new TypedString(sourceJarsResponse))
 
     when:
-    1 * instanceService.getJars() >> {throw new RetrofitError(null, null, null, null, null, null, null)}
+    1 * instanceService.getJars() >> {throw new SpinnakerServerException(request)}
```

Comment:
It would be a smaller change to wrap the RetrofitError in a SpinnakerServerException since that's what SpinnakerRetrofitErrorHandler does.  The exception handling here is simple enough that it probably doesn't matter.

Reply:
Modified the test cases to wrap the Spinnaker*Exception with RetrofitError